src/pages/Aggregator/SlippageSlider/SlippageSlider.component.tsx

```diff
@@ -0,0 +1,34 @@
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Slider from '@mui/material/Slider';
+
+import { SlippageSliderProps } from './SlippageSlider.types';
+
+export const SlippageSlider = ({
```

Comment:
```suggestion
export const Slider = ({
```
The component is general enough so it can be used for any kind of slider, not just for slippage. Rename the rest of the files (`.controlled`, `.types`) as well.